terminal/TerminalClient.cpp

```diff
@@ -130,7 +137,139 @@ void handleWindowChanged(winsize* win) {
   }
 }
 
-int main(int argc, char** argv) {
+void initSetupSSH() {
```

Comment:
Let's move this into a file called SshSetupHandler.hpp/cpp and make initSetupSSH take the passkey, id, jumphost as parameters